### ISSUE TITLE
About Social Force and Netwon 3rd Law

### ISSUE THREAD

**Issue by godisreal** [fij/panic#1]
Do you think the model should be consistent with Newton Laws?   However, the social force does not agree with Newton 3rd Law in its mathematical expression.  
I notice that many of your publications are in the field of physics and currently the model is widely applied in pedestrian motion.  Pedestrian motion definitely is within Newton Laws.  So any comments or insight about this issue?

**Comment by fij** [comment]
The particles (pedestrians) modeled in this case have internal energy depots, and the models of their motion are meant to describe also their decisions on how to use their own energy for steering.

**Comment by godisreal** [comment]
Yes.  I understand your point.   Any creatures like human have energy depots.  We eat and transfer our internal energy into motion.  That is for sure.  However, pedestrian motion is still governed by Newton Laws even if we have such energy depots.  
I suppose the model is consistent with Newton 1st and 2nd Laws.  However, the social force does not agree with Newton 3rd Law (actio=reactio).  That seems a serious problem.  Someone said the particle does not need to follow Newton Laws.  However, pedestrian are low speed object in the macroscopic world, and if social force model is used for pedestrian motion, it should be consistent with Newton Laws.  Do you agree?   
I think that a solution to this problem is that social force is not a physics concept, or we may say it is not a force in physics sense.  Just like the self-driving force, it is realized by the foot-floor friction.  The friction is a physics concept, but the self-driving force is not typically a physics thing.  As you define the "desired velocity" in the model, such desire is in people's mind, and it is not a physics measure.  
As for social force, although f_ij is not equal to f_ji in your model, two people never have any physic force if two do not touch each other physically.  If a pedestrian want to keep a certain distance to others, he or she just moves with feet to adjust the distance.  In other words, we commonly use foot-floor friction to realize the "social force."  So the foot-floor friction is a physics concept, and it is consistent with Newton 3rd Law for interaction between foot and floor.  The social force is not a physics concept, and it is a part of foot-floor friction.  
Now I may get you point about the energy depot.  Yes.  The self-driving force and social force are both from energy depot, and they are consciously controled from our mind, and they are realized by feet-floor friction. Is that a good explanation?
